### ISSUE TITLE
Logspout Logstash TLS issue

### ISSUE THREAD

**Issue by cello86** [gliderlabs/logspout#313]
Hi All,
we tried to move the logspout agent from syslog to syslog+tls implementation, but we have some issue with the self signed certificate present on logstash side.

If we start logspout without CA file specified we received this error:

```
# logspout v3.2-dev by gliderlabs
# adapters: udp tls raw syslog tcp
# options : persist:/mnt/routes
!! x509: certificate signed by unknown authority
```

We restart the docker image with the volume /root/logspout/ca-certs/ related to /mnt/ca/ as reported into the documentation but we noticed a restarting action from the docker daemon and this output:

```
# logspout v3.2-dev by gliderlabs
# adapters: raw syslog tcp udp tls
# options : persist:/mnt/routes
!! EOF 
```

Could we help us to debug the issue?

Thanks,
Marcello

**Comment by gbolo** [comment]
Hi @cello86 

This is because you are using a custom CA to sign your server certificate presented by your syslog server.

This can be addressed by adding your CA cert into the system trust store of logspout server. It can also be addressed in PR #408 where you can use the option: `LOGSPOUT_TLS_CA_CERTS` when this PR gets merged

**Comment by gbolo** [comment]
fixed by #408
